src/test/java/com/znsio/api/jsonPlaceholder.feature

```diff
@@ -0,0 +1,14 @@
+@eat @assignment
+Feature: API tests for https://jsonplaceholder.typicode.com/
```

Comment:
Avoid using URL in Feature description . and rewrite the description to something more meaningful.